engine/src/main/java/com/sandboni/core/engine/Processor.java

```diff
@@ -101,18 +101,12 @@ private Builder getBuilder() {
         return getBuilder(contextSupplier.get());
     }
 
-    /**
-     * Returns true if first: is build stage or runAllExternalTests is false
-     * then: (a) no change was made (b) change was made and contains at least one java file (not just cnfg files)
-     *
-     * @param changeScope the change scope
-     * @return boolean
-     */
     private boolean proceed(ChangeScope<Change> changeScope) {
         return (!isRunAllExternalTests() || !isIntegrationStage())
-            && (arguments.isRunSelectiveMode()
-            || (ChangeScopeAnalyzer.onlySupportedFiles(changeScope, getSupportedFiles())
-            && ChangeScopeAnalyzer.analyzeConfigurationFiles(changeScope, getBuildFiles())));
+                && (arguments.isRunSelectiveMode()
+                || (ChangeScopeAnalyzer.analyzeConfigurationFiles(changeScope, getBuildFiles()))
+                && (arguments.isIgnoreUnsupportedFiles()
+                || ChangeScopeAnalyzer.onlySupportedFiles(changeScope, getSupportedFiles())));
```

Comment:
Did you test it locally with the maven plugin?

Reply:
Yes :)